### PR TITLE
Add Lazy loading to skimage.draw submodule

### PR DESCRIPTION
**Pull request by decorouz**
## Description
Add lazy loading to `skimage.draw` submodule. #6964

## Checklist

- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- Gallery example in `./doc/examples` (new features only)
- Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- Unit tests
- Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)
- Descriptive commit messages (see below)

## For reviewers

- Check that the PR title is short, concise, and will make sense 1 year
  later.
- Check that new functions are imported in corresponding `__init__.py`.
- Check that new features, API changes, and deprecations are mentioned in
  `doc/release/release_dev.rst`.
- There is a bot to help automate backporting a PR to an older branch. For
  example, to backport to v0.19.x after merging, add the following in a PR
  comment: `@meeseeksdev backport to v0.19.x`
- To run benchmarks on a PR, add the `run-benchmark` label. To rerun, the label
  can be removed and then added again. The benchmark output can be checked in
  the "Actions" tab.

### CODE REVIEW

**Review by lagru — APPROVED** [review]
Thanks looks good.

File: skimage/draw/meson.build [hunk 1]
```diff
@@ -8,6 +8,7 @@ py3.extension_module('_draw',
 
 python_sources = [
   '__init__.py',
+  '__init__pyi',
```
Comment (lagru) [inline skimage/draw/meson.build:11]:
```suggestion
  '__init__.pyi',
```

**Comment by lagru** [comment]
By the way @decorouz, your pull request seem to include unnecessary commits. How are you creating these feature branches? The common workflow to create a feature branch is usually:
```sh
git checkout main
git pull upstream main  # Pull new changes 
git checkout -b feature-branch  # Create new feature branch
```

If you ever need to incorporate new changes from `main` while working on a feature branch, I'd advise to do so with:
```sh
# Assumes you are already on the feature branch!
git fetch upstream
git fetch . upstream/main:main  # Fast-forward merge local main without switching branches
git merge main
```

**Comment by lagru** [comment]
Interesting, I can't reproduce the import error locally... :thinking:

**Comment by lagru** [comment]
@decorouz I pushed an attempt to fix this. Please update your local branch with 
```
git checkout lazy_loader_draw_submodule
git pull origin lazy_loader_draw_submodule
```
before you add new changes locally.

**Comment by decorouz** [comment]
@lagru Thanks for the review and fix.

I figured I messed up the flow for incorporating new changes from main. I just didn't know how to fix it. Thanks for the suggestion. I have saved in a note for future reference.

**Review by lagru — APPROVED** [review]
CI is happy now.

**Comment by stefanv** [comment]
Thanks, @decorouz!

**Pull request merged**